MTM-62801 Release notes for cluster handling improvements (#3118)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025.103.0-cluster-handling-improvements.md

---
date:
title: >-
  Cluster membership and node discovery improvements
product_area: Platform services
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62801
version: 2025.101.0
---

> Improvements in the area of handling {{< product-c8y-iot >}} cluster membership and node discover have been implemented that might impact the Microservice monitoring features of the platform.

Reviewer: ```suggestion
Improvements in the area of handling {{< product-c8y-iot >}} cluster membership and node discovery have been implemented that might impact the microservice monitoring features of the {{< product-c8y-iot >}} platform.
```

---

> date: '2025-03-25'

Reviewer: ```suggestion
date:
```

Reviewer: The date will be added automatically on deployment to eu-latest.

title: >-

>   Cluster handling improvements

Reviewer 2: ```suggestion
  Cluster membership and node discovery improvements
```

product_area: Platform services
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62801
version: 2025.103.0
---

> Reliability improvements in handling {{< product-c8y-iot >}} cluster membership changes were implemented affecting the following areas:

Reviewer 2: ```suggestion
Improvements in the area of handling {{< product-c8y-iot >}} cluster membership and node discover have been implemented that might impact the Microservice monitoring features of the platform.
```

Reviewer 3: in this case version changes to `2025.101`

>  - Microservice monitoring

Reviewer 2: ```suggestion
```

>  - microservice monitoring

Reviewer: ```suggestion
 - Microservice monitoring
```

>  - Data broker

Reviewer 2: ```suggestion
```

>  - data broker

Reviewer: ```suggestion
 - Data broker
```

>  - Cache invalidation

Reviewer 2: ```suggestion
```

>  - cache invalidation

Reviewer: ```suggestion
 - Cache invalidation
```

>  - Platform certificate updates

Reviewer 2: ```suggestion
```

>  - platform certificate updates

Reviewer: ```suggestion
 - Platform certificate updates
```
